### PR TITLE
Expose RootCloseWrapper as part of public API

### LINKED ISSUE #77: Include RootCloseWrapper as part of public API
`RootCloseWrapper` is small but definitely useful on its own. What are your thoughts about adding it as part of the public API?

### PR DESCRIPTION
**Pull request by aatrostle**
This PR exposes `<RootCloseWrapper/>` as part of the public API and adds examples. Resolves #77.

Thanks to the maintainers, this package has been super helpful ❤️

### CODE REVIEW

File: src/RootCloseWrapper.js [hunk 1]
```diff
@@ -15,7 +15,11 @@ function isModifiedEvent(event) {
   return !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
 }
 
-export default class RootCloseWrapper extends React.Component {
+/**
+ * The `<RootCloseWrapper/>` registers your callback on the document when rendered. Powers the
```
Comment (taion) [inline src/RootCloseWrapper.js]: worth noting this also triggers on users hitting `esc`

Comment (aatrostle) [inline src/RootCloseWrapper.js]: Should I note that here in the desc or in the `onRootClose` prop?

Comment (taion) [inline src/RootCloseWrapper.js]: possibly both – and also describe the purpose of root close wrapper (i.e. to make popovers behave in a modal manner where they close when the user tries to interact with the rest of the doucment)

**Comment by aatrostle** [comment]
@taion Review feedback addressed 👍

**Comment by jquense** [comment]
LGTM

**Pull request merged**